.github/actions/high-priority-prs/src/pr-message.js

```diff
@@ -57,8 +57,8 @@ module.exports = (queues, maintainers, now = new Date()) => {
           : ""
 
       // console.log({ participated })
-      const createdAgo = distanceInWords(parse(pr.createdAt), now)
-      const updatedAgo = distanceInWords(parse(pr.updatedAt), now)
+      const createdAgo = formatDistance(parseISO(pr.createdAt), now)
+      const updatedAgo = formatDistance(parseISO(pr.updatedAt), now)
```

Comment:
Per https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#200---2019-08-20 :

> The order of arguments of `distanceInWords` and `distanceInWordsStric`t is swapped to make them consistent with `differenceIn...` functions.

so we might need to switch args order here